src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -2169,10 +2172,10 @@ MeleeHitOutcome Unit::RollMeleeOutcomeAgainst(Unit const* victim, WeaponAttackTy
     }
 
     // 4. GLANCING
-    // Max 40% chance to score a glancing blow against mobs that are higher level (can do only players and pets and not with ranged weapon)
+    // Max 40% chance to score a glancing blow against mobs that are same or higher level (can do only players and pets and not with ranged weapon)
```

Comment:
Comment readability improvement suggestion:
```suggestion
    // Max 40% chance to score a glancing blow against mobs of the same or higher level (only players and pets, not for ranged weapons).
```
(Feel free to phrase it differently, in case this suggestion misses any of the intended descriptions.)

Reply:
Thanks, it's clearer.